.github/workflows/sanitized_ci.yml

```diff
@@ -145,13 +145,15 @@ jobs:
           ./v retry -- sudo apt install clang
       - name: Recompile V with -cstrict
         run: ./v -cg -cstrict -o v cmd/v
+      - name: Create LSAN suppression file for ignoring websocket test
+        run: echo "leak:mbedtls_ssl_setup" > suppress.txt
```

Comment:
imho add it as a file in the repo instead of creating it inline here, see for example the existing `.github/workflows/run_sanitizers_thread.suppressions` and `.github/workflows/run_sanitizers_undefined.suppressions` .

Reply:
Oh great.